test/validate/windows_validate.go

```diff
@@ -13,12 +14,22 @@ import (
 	"k8s.io/client-go/rest"
 )
 
+const (
+	cnsWinLabelSelector = "k8s-app=azure-cns-win"
+)
+
 var (
-	hnsEndPointCmd      = []string{"powershell", "-c", "Get-HnsEndpoint | ConvertTo-Json"}
-	hnsNetworkCmd       = []string{"powershell", "-c", "Get-HnsNetwork | ConvertTo-Json"}
-	azureVnetCmd        = []string{"powershell", "-c", "cat ../../k/azure-vnet.json"}
-	azureVnetIpamCmd    = []string{"powershell", "-c", "cat ../../k/azure-vnet-ipam.json"}
-	restartKubeProxyCmd = []string{"powershell", "Restart-service", "kubeproxy"}
+	hnsEndPointCmd                 = []string{"powershell", "-c", "Get-HnsEndpoint | ConvertTo-Json"}
```

Comment:
Did we try using `-AsArray` for the ConvertTo-Json cmd ? Will that help us to avoid a switch case for object and array while reading the data.
ref: https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/convertto-json?view=powershell-7.4#-asarray

Reply:
It is currently not possible as our daemonset is using Powershell 5.1. When trying to update the image being used by the ds for a newer version of Powershell it would still use 5.1. Further effort is required to understand what is preventing the use of 7.*

Reply:
Not a blocking comment. So lgtm.